docs/section10/10-9.md

```diff
@@ -0,0 +1,51 @@
+# Airside systems – Rule 10-9  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** True     
+**Rule ID:** 10-9  
+ 
+**Rule Description:** Where multiple HVAC zones or residential spaces are combined into a single thermal block, baseline HVAC System Types 5 or 6 efficiencies shall be based on the cooling equipment capacity of a single floor.  
+
+**Rule Assertion:** B-RMR = expected value                                             
+**Appendix G Section:** Section G3.1.2.1  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. Applicable to baseline HVAC system types 5 or 6 that include more than one floor.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()
+2. get_hvac_systems_5_6_serving_multiple_floors_b()
+
+**Manual Check:** None.  
+ 
+## Rule Logic:  
+**Applicability Check 1:**  
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Get dictionary with hvac system ids serving more than one floor as keys and the number of floors served as values: `hvac_sys_serve_more_than_one_flr_dict_b = get_hvac_systems_5_6_serving_multiple_floors_b(B_RMR)`
+- Get list of hvac systems of type 5 and 6 that serve more than one floor: `hvac_sys_serve_more_than_one_flr_list_b = hvac_sys_serve_more_than_one_flr_dict_b.keys()`
+- For each hvac_b in hvac_sys_serve_more_than_one_flr_list_b: `for hvac_b in hvac_sys_serve_more_than_one_flr_list_b:`
+    - Get the hvac system type for hvac_b: `hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)]`
+    - Check if HVAC system is type 5, 5b, 6, 6b, if it is then carry on: `if get_baseline_system_types(hvac_b.id) in ["SYS-5", "SYS-5b", "SYS-6", "SYS-6b"]:`
```

Comment:
Lines 31 and 32 seems redundant because function only returns those system types

Reply:
Revised